popup.js

```diff
@@ -72,6 +83,7 @@ function doItAll() {
   }
 
   $("#addToList").click(function () {
+    doItAll
```

Comment:
I don't know why this works ... I guess you don't need the "( )" in a function call ... 
But still, this should not be here